index.js

```diff
@@ -9,8 +10,17 @@ var isGzipped = function(data) {
   return true
 }
 
-module.exports = function() {
+var createParser = function() {
+  var zlibGunzip = zlib.createGunzip()
+  var gunzipObj = gunzip()
+  zlibGunzip.pipe(gunzipObj)
+  return duplexify(zlibGunzip, gunzipObj)
+}
+
```

Comment:
Makes a lot of sense. Will adjust.

I'm still shocked with the useful and diverse stream arsenal toolkit you have.